civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -10,33 +10,88 @@ use Drupal\civihr_employee_portal\Helpers\HelperClass;
  */
 function civihr_employee_portal_css_alter(&$css) {
 
-    // Remove defaults system messages css file.
-    unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+  // Remove defaults system messages css file.
+  unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+
+  // Removing unwanted css files from civicrm pages.
+  if (civicrm_invoke()) {
+    // Multi-dimensional array containing module name and file path.
+    $removeFiles = [
+      ['module_name' => 'ctools', 'file_name' => '/css/ctools.css'],
+      ['module_name' => 'date_api', 'file_name' => '/date.css'],
+      [
```

Comment:
If you are having all the module_name/file_name duplets inline, please keep also this one inline as well